Clinical-Genomics/scout
# Use start+length as end for insertions when querying loqusdb. Fix #1743

This PR fixes a problem with local frequencies for INSertions. See #1743 

**Prepare for tests**:
1. - [x] Go to a case with a known insertion. Check that there are no local observations:
![Skärmavbild 2020-02-21 kl  15 16 52](https://user-images.githubusercontent.com/405278/75041651-4d42d880-54bd-11ea-93d5-3533f8db7c3f.png)

**How to test**:
1. - [x] Install on clinical-db
1. - [x] Refresh and check that there are observations
1. - [x] Upload a screendump

**Expected outcome**:
The observations should be shown

**Review:**
